profiles/tests/test_gql_service_connection_with_user_id_query.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+
+from graphql_relay import to_global_id
+
+QUERY = """
+    query ($userId: UUID!, $serviceClientId: String!)
+    {
+        serviceConnectionWithUserId(
+            userId: $userId, serviceClientId: $serviceClientId
+        ) {
+            id
+            createdAt
+            enabled
+            service {
+                id
+                name
+                title
+                description
+                createdAt
+                gdprUrl
+                gdprQueryScope
+                gdprDeleteScope
+            }
+        }
+    }
+"""
+
+
+def assert_datetimes_equal(datetime_str, datetime_instance):
```

Comment:
Could this be in the open_city_profile.tests.asserts module?

Reply:
It's a bit special assert because it requires the first argument to be a string and the second to be a datetime. That's why I didn't consider it to be very reusable and didn't include it to the common asserts.